ui/src/app/components/sidenav-view-assessment/sidenav-view-assessment.component.html

```diff
@@ -1,2 +1,101 @@
-<h2>Assessment content coming soon!</h2>
-<button mat-button (click)="closeSidenav()">Close</button>
+<div>
+  <div class="header">
+    <span class="mat-h3 header-title">Assessment report</span>
+    <button class="close-button" mat-icon-button color="primary" (click)="closeSidenav()">
+      <mat-icon class="close-icon">close</mat-icon>
+    </button>
+  </div>
+  <div class="content">
```

Comment:
There is no left spacing between the dialog box and contents. The margin/padding needs to be fixed as in other components

Reply:
Updated the CSS